cli/consts.py

```diff
@@ -119,3 +119,14 @@
 
 SKIP_SCAN_FLAG = 'skip-cycode-scan'
 VERBOSE_SCAN_FLAG = 'verbose'
+
+LICENSE_COMPLIANCE_POLICY_ID = '8f681450-49e1-4f7e-85b7-0c8fe84b3a35'
+PACKAGE_VULNERABILITY_POLICY_ID = '9369d10a-9ac0-48d3-9921-5de7fe9a37a7'
```

Comment:
why we need policy ids here?

Reply:
There is no other way to distinguish between policy types.